x-pack/plugins/security_solution/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -50,11 +50,13 @@ export const signalRulesAlertType = ({
   version,
   ml,
   lists,
+  errorService,
 }: {
   logger: Logger;
   version: string;
   ml: SetupPlugins['ml'];
   lists: SetupPlugins['lists'] | undefined;
+  errorService: SetupPlugins['errorService'];
```

Comment:
IMO `errorService is a bit of a misnomer here, this is just our own savedObjectsClient. I'd call it `internalSavedObjectsClient` myself.